image-builder/README.md

```diff
@@ -0,0 +1,68 @@
+# Image Builder Research
+
+We would like to use Image Builder to compose bootable images that would
+contain upstream code that anyone can boot and play with it, especially
+contributors.
+
+Packit needs to work with Image Builder API in order to start those image builds.
+
+## Image Builder API
+
+[@ondrejbudai](https://github.com/ondrejbudai) created [a great intro blog
+post](https://hackmd.io/Lrbf_6Q9SZy06iEj5S5Wuw) how to use the API. Please read
+it before continuing reading.
+
+TL;DR
+
+1. Create a token [here](https://access.redhat.com/management/api) and generate
+   an access token from it to use with Image Builder API
+2. API docs: https://github.com/osbuild/image-builder/blob/main/internal/v1/api.yaml
+3. Web UI: https://console.redhat.com/insights/image-builder/
+4. [Image Builder service docs](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/uploading_a_customized_rhel_system_image_to_cloud_environments/creating-a-customized-rhel-system-image-for-aws-using-image-builder)
+5. The team has their own gchat room, guess the name :)
+
+## Drawbacks
+
+The API works well, but the experience wasn't perfect. Be aware of these:
+
+1. There are no build logs and there is no plan right now to have them.
+2. In case of failure, there is only a single error message provided that can
+   be confusing (see Jira tickets linked from
+   https://issues.redhat.com/browse/PACKIT-1940 to HMSIB)
+3. No callbacks, one needs to poll to find out about the completion about an
+   image build
+
+## PoPoC
+
+We have a PoC in mind (Installer team needs bootable images off PRs to see the
+change in action). I started a simple python script that submits an image build
+and waits for it to finish. The image installs cockpit on RHEL 9 from cockpit's
+releases Copr project.
+
+Once the image is built, there is a link in the webui to launch an AWS EC2
+instance and play with it. Neat!
+
+### Usage
+
+Set the offline token via `REDHAT_API_OFFLINE_TOKEN` env var and run the
+"image_build.py" script.
+
+## Next steps
+
+The integration is pretty straightforward:
+
+1. Create a new job: "image-builder" (or "image-build"?)
```

Comment:
For me, "image" automatically means "a container image".
When I started reading the README, I was confused about why "bootable", until I realized we're talking about VM images here.
Also, if we wanted to produce container images one day, we would need to distinguish the jobs somehow. Therefore I suggest "vm-image-build" in this case.

And +1 to `build`, not `builder`.